sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1777,7 +1777,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
           """Extended Usage:
             |    Examples:
             |      > SELECT 3 ^ 5;
-            |       2
+            |       6
```

Comment:
Oh haha . I was about to say we shouldn't test this. but seems like single case to test Extended usage. Looks fine then.

Reply:
Seriously I wouldn't expect fixing doc makes some _test case_ failed too.

Reply:
Ya... Sorry for this trouble. :(

Reply:
Do we need to add `[HOT-FIX]` to the title because other tests depends on this pr:

https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/105669/testReport/org.apache.spark.sql.hive.execution/HiveCatalogedDDLSuite/describe_function/

Reply:
Let's do that.